packages/app-page-builder/src/admin/views/BlockCategories/DelayedOnChange.ts

```diff
@@ -0,0 +1,107 @@
+import * as React from "react";
```

Comment:
@Pavel910 there are few places where such component is used:
![image](https://user-images.githubusercontent.com/77202393/186455079-38572427-70d8-4dbe-95be-a04d9d664976.png)
But there are 2 variants: as a class component and as a functional component.
I am not 100% sure they are completely same (but they looks similar).

Which variant is preferable for extraction? Should I replace other usages too?

Reply:
@neatbyte-vnobis please use the functional component variant, and only do it for the page builder package; we'll take care of other packages, just to avoid running into rabbit holes with this PR :) Thanks!